DirectProgramming/DPC++FPGA/Tutorials/Features/scheduler_target_fmax/README.md

```diff
@@ -0,0 +1,147 @@
+# Scheduler Target FMAX
+This tutorial explains the `scheduler_target_fmax_mhz` attribute and its effect on the performance of Intel® FPGA kernels.
+
+***Documentation***:  The [DPC++ FPGA Code Samples Guide](https://software.intel.com/content/www/us/en/develop/articles/explore-dpcpp-through-intel-fpga-code-samples.html) helps you to navigate the samples and build your knowledge of DPC++ for FPGA. <br>
+The [oneAPI DPC++ FPGA Optimization Guide](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide) is the reference manual for targeting FPGAs through DPC++. <br>
+The [oneAPI Programming Guide](https://software.intel.com/en-us/oneapi-programming-guide) is a general resource for target-independent DPC++ programming.
+
+| Optimized for                     | Description
+---                                 |---
+| OS                                | Linux* Ubuntu* 18.04/20.04, RHEL*/CentOS* 8, SUSE* 15; Windows* 10
+| Hardware                          | Intel® Programmable Acceleration Card (PAC) with Intel Arria® 10 GX FPGA <br> Intel® FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix® 10 SX) <br> Intel® FPGA 3rd party / custom platforms with oneAPI support <br> *__Note__: Intel® FPGA PAC hardware is only compatible with Ubuntu 18.04*
+| Software                          | Intel® oneAPI DPC++ Compiler <br> Intel® FPGA Add-On for oneAPI Base Toolkit
+| What you will learn               |  The behavior of the `scheduler_target_fmax_mhz` attribute and when to use it on your kernel <br> The effect this attribute can have on your kernel's performance on FPGA
+| Time to complete                  | 15 minutes
+
+## Purpose
+This tutorial demonstrates how to use `[[intel::scheduler_target_fmax_mhz(N)]]` attribute to direct the compiler to prioritize a high fMAX over a low II.
+
+A bottleneck in a loop is one or more loop-carried dependencies that cause the loop to have either an II greater than one, or a lower fMAX to achieve II of one. By default, the compiler finds the fMAX-II combination that provides the best fMAX-to-II ratio.
+
+The compiler provides `[[intel::scheduler_target_fmax_mhz(N)]]` attribute to set your desired fMAX. When you specify the fMAX, the compiler schedules the design at that specified fMAX value with minimal achievable II.
```

Comment:
*to set your desired fMAX for a given kernel.

Reply:
Should we also include the `-Xs` flag as well to show that you can set the global target as well in this tutorial?

Reply:
> *to set your desired fMAX for a given kernel.

Will do.



> Should we also include the `-Xs` flag as well to show that you can set the global target as well in this tutorial?

Good point. I can use the `-Xsclock` flag in this tutorial. It will also demonstrate the kernel attribute overrides the global flag.

Reply:
Yep - sounds good to me. Just make to sure to mention that the attribute takes precedence.